group_members.go

```diff
@@ -176,6 +176,31 @@ func (s *GroupMembersService) AddGroupMember(gid interface{}, opt *AddGroupMembe
 	return gm, resp, err
 }
 
+// ShareWithGroup shares a group with the group.
+//
+// GitLab API docs:
+// https://docs.gitlab.com/ce/api/groups.html#share-groups-with-groups
+func (s *GroupMembersService) ShareWithGroup(gid interface{}, opt *ShareWithGroupOptions, options ...RequestOptionFunc) (*Group, *Response, error) {
```

Comment:
Is `ShareWithGroupOptions` defined anywhere already? Or did you forgot to commit that?

Reply:
ShareWithGroupOptions already exists for projects: https://github.com/xanzy/go-gitlab/blob/f2b3cfead604678820f486a362a1b264895589f3/projects.go#L791

It should be able to be reused, or duplicated for sharing groups with groups